ui/package.json

```diff
@@ -16,11 +16,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint",
-      "pre-push": "npm run lint",
-      "...": "..."
```

Comment:
Add it for pre-push too.

Reply:
It isn't supported for pre-push. An ongoing discussion here- https://github.com/okonet/lint-staged/issues/303

Reply:
Cool!